backend/app/services/adapters/executor_kinds.py

```
@@ -493,6 +493,40 @@ def _format_subtasks_response(
+                                        if (
+                                            "env" in private_model_config
+                                            and "api_key" in private_model_config["env"]
+                                        ):
+                                            private_model_config["env"]["api_key"] = decrypt_api_key(
+                                                private_model_config["env"]["api_key"]
+                                            )
+                                        agent_config_data = private_model_config
+                                        logger.info(
+                                            f"Successfully loaded legacy private_model config: {private_model_name.strip()}"
+                                        )
+                            except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)